### DISCUSSION TITLE
Is there currently a working twitter endpoint that returns the latest tweets by a user that does not require auth?

### DISCUSSION THREAD

**Discussion by s0ph33** [fa0311/twitter-openapi#82] [General]
I am looking for an unauthed endpoint that returns a user's latest tweet or tweets. I believe one exists but so far I've been unable to find it. Would be grateful if someone can share an endpoint or at least point me in the right direction

**Comment by fa0311** [comment]
Unfortunately, no.
All have been deleted.

**Reply by s0ph33** [reply]
:( How about an authed one with decent rate limits? Different one than UserTweets, ListLatestTweetsTimeline, etc... There was a decent v1 endpoint for lists that got disabled today

**Reply by fa0311** [reply]
As far as I know it does not exist.
